wmn-data.json

```diff
@@ -5455,6 +5527,30 @@
         "cat" : "blog",
         "valid" : true
        },
+       {
+        "name" : "StreamElements",
+        "uri_check" : "https://api.streamelements.com/kappa/v2/channels/{account}",
+        "post_body" : "",
+        "e_code" : 200,
+        "e_string" : "username",

```

Comment:
Can you choose a different `e_string` that is more complex? Something like `\"username\" would work or `providerId`